spec/models/campaign_spec.rb

```diff
@@ -0,0 +1,137 @@
+require 'rails_helper'
+
+RSpec.describe Campaign, type: :model do
+  let(:campaign) { build(:campaign) }
+
+  context 'when title field is empty' do
```

Comment:
you have added https://github.com/thoughtbot/shoulda-matchers so please rewrite all tests as we agreed